e107_admin/userclass2.php

```diff
@@ -718,16 +718,16 @@ function checkAllowed($classID, $redirect = true, $fullEdit = FALSE)
 		//		unset($pref['frontpage'][$class_id]);		// (Should work with both 0.7 and 0.8 front page methods)
 		//		save_prefs();
 			}
-			$emessage->add(UCSLAN_3, E_MESSAGE_SUCCESS);
+			$emessage->add(UCSL/AN_3, E_MESSAGE_SUCCESS);
```

Comment:
Stops old lans in commented blocks from falsely appearing as missing must be renabled in deprecated LANs checker. My bad?